NitroxLauncher/Pages/ServerPage.xaml.cs

```diff
@@ -405,7 +427,7 @@ private void TBWorldSeed_Input(object sender, KeyboardFocusChangedEventArgs e)
             Config.Seed = TBWorldSeed.Text;
         }
 
-        private void TBMaxPlayerCap_Input(object sender, System.Windows.Input.KeyboardFocusChangedEventArgs e)
+        private void TBMaxPlayerCap_Input(object sender, KeyboardFocusChangedEventArgs e)
```

Comment:
Unsure why, but I think the cleanup profile may have done these changes removing the specific import path. I can add these back if this layout is preferred

Reply:
World manager was developed by one of our support team (non-developer), so this probably due to adding event with XAML editor